enclave-runner/src/usercalls/mod.rs

```diff
@@ -545,6 +556,50 @@ fn trap_attached_debugger(tcs: usize) {
     }
 }
 
+/// `UsercallExtension`
+/// Provides a mechanism for the enclave code interface with an external service via a modified runner.
```

Comment:
This isn't quite what I meant with adding an empty line. But I now see this needs additional rewording. Do this:

```patch
-/// `UsercallExtension`
-/// Provides a mechanism for the enclave code interface with an external service via a modified runner.
+/// Provides a mechanism for the enclave code to interface with an external service via a modified runner.
+///
```

Reply:
Thanks. Fixed in the recent push.

Reply:
You didn't make the exact change I suggested.

Reply:
Got it. I missed the empty comment instead I used empty newline. Pushed a fix.

Reply:
You still didn't delete the line that just says
```
/// `UsercallExtension`
```

Reply:
Fixed in recent push.